README.md

```diff
@@ -35,6 +43,24 @@ the build to fail.
 *NOTE:* detecting the base version only works if you have git tags in
 the SemVer-compliant `x.y.z` format, such as `1.2.3`.
 
+#### Github action
+
+You can use it as a Github Action like this:
+
+_.github/main.workflow_
+```
+workflow "Main" {
+  on = "push"
+  resolves = ["Roave BC Check"]
+}
+
+action "Roave BC Check" {
+  uses = "docker://nyholm/roave-bc-check-ga"
+  secrets = ["GITHUB_TOKEN"]
```

Comment:
I think the secret is not needed. I did this for my github actions too, I didn’t write or use any github api endpoints in the image